### PR TITLE
Make test tools react more humanly

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

For now, master says 0 is not 1. Attaching helper description along with other assertion improvements so each step can be debugged easier

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/notificationservice/RestAssuredHelper.java [hunk 1]
```diff
@@ -45,7 +45,9 @@ static String s2sSignIn(String microservice) {
             .post("/lease")
             .andReturn();
 
-        assertThat(response.getStatusCode()).isEqualTo(200);
+        assertThat(response.getStatusCode())
+            .as("OTP leased successfully")
```
Comment (rkondratowicz) [inline src/functionalTest/java/uk/gov/hmcts/reform/notificationservice/RestAssuredHelper.java]: otp is sent to this endpoint, not received from it.

Comment (rkondratowicz) [reply src/functionalTest/java/uk/gov/hmcts/reform/notificationservice/RestAssuredHelper.java]: what we get here is s2s token I think

Comment (doncem) [reply src/functionalTest/java/uk/gov/hmcts/reform/notificationservice/RestAssuredHelper.java]: lease means contract or some sort of grant. in super short way. so looks like sentence is ok. feel free to suggest any sentence that may be suitable/more understandable for us :slightly_smiling_face:

Comment (rkondratowicz) [reply src/functionalTest/java/uk/gov/hmcts/reform/notificationservice/RestAssuredHelper.java]: method is called `s2sSignIn` so maybe 'Signed in to S2S successfully'.

**Pull request merged**